src/app/build/build.component.scss

```diff
@@ -1,16 +1,24 @@
 @import 'global_variables';
 @import 'global_states';
 
+.build-chart-bg {
+  background: linear-gradient(5deg, darken(#F0F1F6, 2.5%) 0%, lighten(#e6e9f0, 1%) 100%);
+  display: block;
+  content: " ";
+  top: 0px;
```

Comment:
Nit: can we be consistent and use em/percentages? Or for zeroes use 0? Not blocking whatsoever, just sayin'.